sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md

```diff
@@ -2,13 +2,8 @@
 
 ## 4.10.0-beta.1 (Unreleased)
```

Comment:
Should the version and date be updated as well or will it be done in a separate PR?

Reply:
It will be done in the separate PR

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/27568